jinahub/encoders/audio/AudioCLIPEncoder/audio_clip/utils/simple_tokenizer.py

```diff
@@ -11,7 +11,7 @@
 
 @lru_cache()
 def default_bpe():
-    return os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'assets', 'bpe_simple_vocab_16e6.txt.gz')
+    return os.path.join(os.getcwd(), '.cache', 'bpe_simple_vocab_16e6.txt.gz')
```

Comment:
Why do we rely on `cwd`? I prefer it the old way

Reply:
I will change it to use `__file__`